treeherder/autoclassify/matchers.py

```diff
@@ -165,11 +165,6 @@ class ElasticSearchTestMatcher(Matcher):
     """Matcher that looks for existing failures with identical tests, and error
     message that is a good match when non-alphabetic tokens have been removed."""
 
-    def __init__(self, *args, **kwargs):
-        Matcher.__init__(self, *args, **kwargs)
-        self.lines = 0
-        self.calls = 0
```

Comment:
I think this is used by:
https://github.com/mozilla/treeherder/blob/1dd0fdf11e48f21fd750625d668e1fe2d2d76703/treeherder/autoclassify/management/commands/es_benchmark.py#L92-L93

Though there may be a better way to do that.

Reply:
I've put the `.calls` variable back in but removed lines from the benchmark script since it's never updated in the class anymore.  I'll have a look at the benchmark script properly another time, I'm not sure if it's still relevant to the current ES code.